server/lambda/phacToSbc/index.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-unresolved */
+const { sendPhacToSBC } = require('custom_modules/sbc-phac-ets');
+const dbConnectionAndCollections = require('custom_modules/db');
+const markDuplicates = require('./mark-duplicates');
+
+/* eslint-disable no-console */
+
+exports.handler = async () => {
+  const { connection, collections } = await dbConnectionAndCollections(['ets-forms', 'ets-phac']);
+  const [etsCollection, phacCollection] = collections;
+  try {
+    const duplicates = await markDuplicates(etsCollection, phacCollection);
+    console.log(duplicates);
+    const transactions = await sendPhacToSBC(phacCollection);
+    console.log(transactions);
+  } catch (error) {
+    console.error(`Failed to mark duplicates or post to SBC ${error}`);
```

Comment:
I feel like we could get burned by this in future, depending on how specific the error messages are. We will really want to be able to answer the question of whether de-duplication failed or whether the send to SBC failed. Should we split this into two separate try/catch blocks to play it safe, in case the error isn't informative enough to make the distinction?

Reply:
As far as I understood the specific error object will be printed out whether is a connection issue or database, but I'm not sure if doing that as console.error() in the lambda code will print in any CloudWatch group, probably not.

Reply:
My concern is more around whether we'll be able to distinguish at which stage the error occurred, like did the DB go offline during sendPhacToSBC vs. during duplication? But I think it might be unfounded, actually, since in that case we would have logs from the de-duplication script and be able to see that it completed prior to the error